pom.xml

```diff
@@ -59,9 +59,9 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <cdap.version>5.0.0</cdap.version>
```

Comment:
should we also change cdap version to be the latest?

Reply:
doesn't really matter since nothing's changed